contracts/examples/crowdfunding-erc20/mandos/deploy_erc20_and_crowdfunding.scen.json

```diff
@@ -1,131 +1,130 @@
+            "expect": {
+                "status": "0",
+                "gas": "*",
+                "refund": "*"
+            }
+        },
+        {
+            "step": "checkState",
+            "accounts": {
+                "address:erc20_owner": {
+                    "nonce": "1",
+                    "balance": "0",
+                    "storage": {}
+                },
+                "address:crowdfunding_owner": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {}
+                },
+                "sc:erc20": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {
+                        "str:balance|address:erc20_owner": "1,000,000,000",
+                        "str:totalSupply": "1,000,000,000"
+                    },
+                    "code": "file:../../erc20/output/erc20.wasm"
+                }
+            }
+        },
+        {
+            "step": "scDeploy",
+            "txId": "deploy",
+            "tx": {
+                "from": "address:crowdfunding_owner",
+                "contractCode": "file:../output/crowdfunding-erc20.wasm",
+                "value": "0",
+                "arguments": [
+                    "1,000,000",
+                    "123,456",
+                    "sc:erc20"
+                ],
+                "gasLimit": "50,000,000",
+                "gasPrice": "0"
             },
-            "code": "file:../output/crowdfunding-erc20.wasm"
-          }
+            "expect": {
+                "status": "0",
+                "gas": "*",
+                "refund": "*"
+            }
+        },
+        {
+            "step": "checkState",
+            "accounts": {
+                "address:erc20_owner": {
+                    "nonce": "1",
+                    "balance": "0",
+                    "storage": {}
+                },
+                "address:crowdfunding_owner": {
+                    "nonce": "1",
+                    "balance": "0",
+                    "storage": {}
+                },
+                "sc:erc20": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {
+                        "str:balance|address:erc20_owner": "1,000,000,000",
+                        "str:totalSupply": "1,000,000,000"
+                    },
+                    "code": "file:../../erc20/output/erc20.wasm"
+                },
+                "sc:crowdfunding": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {
+                        "str:target": "1,000,000",
+                        "str:deadline": "123,456",
+                        "str:erc20ContractAddress": "sc:erc20"
+                    },
+                    "code": "file:../output/crowdfunding-erc20.wasm"
+                }
+            }
         }
-      }
     ]
-  }
-  
```

Comment:
no newline at end of file?

Reply:
It has. I think github got confused. The script makes sure they all end in newlines